domains/sampath.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "gujarathisampath",
+           "email": "",
+           "discord": "984015688807100419",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.pl6QsQtuSeTxI4qW8VNCuAxSYrmPySIp-thnTbJzhdXyYrejfz3ypbCWqYX02RusfAkil6PMI_Y59cKhi_-EqWsEPNywQOqS1YWOQdRcH3XBIbZOXolGYyOUsCzCJDt52bn3RMbU0v1C7nw7bBElgOq6oQNYmUtQBPSG3BrEDJnwtyaIWAQmrI19mdG2lgMITPtB6bc9lkyDphg5CN44VON-dflwPzajVLcLKd7UFn-aupC8_k0SQ2Ls-9ldRFe45yrS2I1rK5NA5CdXahVn0HqOR4hjYBAcZ9n9TDWOdIw2KNKVy9uiKeS5omruwDaJlUX_rV6dZ0IYL4TFmRFqRQ.sdhuYH6WZA070TY-hK14_w.oNx0grctR0mFrgxZsrG_6alCz2x_XmYCWpot7ZEDP7HPaBu4WMLzUXAOX1e6yOE4UJX2Fuu3woZGmE9LHibD6KGw5fM6EHG0TFFzGbHdqh1S5P1OcGRVT46wEFMx8vh7.R4PT633lIs8sA7pkAaS8mg"
+        },
+    
+        "record": {
+            "CNAME": "cname.vercel-dns.com"
```

Comment:
Can you give a preview of your website?

Reply:
![image](https://github.com/user-attachments/assets/8bca877a-7684-4962-9a2e-319b28e44ef3)